CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [11.2.0] - UPCOMING
 
 * UPCOMING
```

Comment:
Please remove this bullet, replacing it with yours.

Reply:
Done.